doc/users/whats_new/reproducible_ps_pdf.rst

```diff
@@ -1,14 +1,26 @@
-Reproducible PS and PDF output
-------------------------------
+Reproducible PS, PDF and SVG output
+-----------------------------------
 
 The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
 the timestamp value in the PS and PDF outputs. See
 https://reproducible-builds.org/specs/source-date-epoch/
 
+Alternatively, calling ``savefig`` with ``metadata={creationDate=None}``
```

Comment:
This `dict` literal is a bit broken; should be `dict(creationDate=None)` or `{'creationDate': None}`.